main/res/values/strings.xml

```diff
@@ -1287,6 +1287,7 @@
     <string name="map_as_list">Show as list</string>
     <string name="map_select_multiple_items">Select item</string>
     <string name="map_routing">Routing</string>
+    <string name="map_routing_brouter">Routing is only available if <a href="">brouter</a> is installed</string>
```

Comment:
LGTM, thanks for implementation. Just a tiny thing: The app is named "BRouter", or "BRouter Offline Navigation" (though I would stay with the short form) - can you adapt this, please?

Reply:
Done!